data/wikipron/src/languages.json

```diff
@@ -693,7 +693,11 @@
         "iso639_name": "Mongolian",
         "wiktionary_name": "Mongolian",
         "wiktionary_code": "mn",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "cyrl": "Cyrillic",
+            "mong": "Mong"
```

Comment:
Since it looks like there's not nearly enough of this you could remove the Mong line?

Reply:
Sure. Removed the old script from the registry.